glancesync/glancesync_serversfacade.py

```diff
@@ -105,7 +148,29 @@ def upload_image(self, regionobj, image):
         :param image: GlanceSyncImage object; the image to be uploaded.
         :return: The UUID of the new image.
         """
-        return _upload_image(regionobj, image, self.images_dir)
+        client = self._get_glanceclient(regionobj.region)
+        try:
+            with open(self.images_dir + '/' + image.id, 'r') as file_obj:
+                try:
+                    new_image = client.images.create(
+                        container_format=image.raw['container_format'],
+                        disk_format=image.raw['disk_format'],
+                        name=image.name, is_public=image.is_public,
+                        protected=image.raw['protected'],
+                        min_ram=image.raw['min_ram'],
+                        min_disk=image.raw['min_disk'],
+                        properties=image.user_properties, data=file_obj)
+                    return new_image.id
+                except Exception, e:
+                    msg = regionobj.fullname + ': Upload of ' + image.name +\
```

Comment:
Let me see, I see at least twice this message, why not define a string like this

xxx = '{}: Upload of {} Failed. Cause: {}'

And then just invoke the .format with the parameters in the self.logger.error sentence for example?